src/renderer/components/watch-video-info/watch-video-info.vue

```diff
@@ -1,79 +1,84 @@
 <template>
-  <ft-card class="relative watchVideoInfo">
-    <p
-      class="videoTitle"
-    >
-      {{ title }}
-    </p>
-    <div
-      class="channelInformation"
-    >
-      <img
-        :src="channelThumbnail"
-        class="channelThumbnail"
-        @click="goToChannel"
+  <ft-card class="watchVideoInfo">
+    <div>
+      <p
+        class="videoTitle"
       >
-      <span
-        class="channelName"
-        @click="goToChannel"
+        {{ title }}
+      </p>
+      <div
+        class="channelInformation"
       >
-        {{ channelName }}
-      </span>
-      <ft-button
-        :label="subscribedText"
-        class="subscribeButton"
-        background-color="var(--primary-color)"
-        @click="handleSubscription"
-      />
+        <div
+          class="profileRow"
+        >
+          <div>
+            <img
+              :src="channelThumbnail"
+              class="channelThumbnail"
+              @click="goToChannel"
+            >
+          </div>
+          <div>
+            <div
+              class="channelName"
+              @click="goToChannel"
+            >
+              {{ channelName }}
+            </div>
+            <ft-button
+              :label="subscribedText"
+              class="subscribeButton"
+              background-color="var(--primary-color)"
+              @click="handleSubscription"
+            />
+          </div>
+        </div>
+      </div>
     </div>
-    <ft-flex-box class="videoOptions">
-      <ft-icon-button
-        title="Toggle Theatre Mode"
-        class="theatreModeButton"
-        icon="expand-alt"
-        theme="secondary"
-        @click="$emit('theatreMode')"
-      />
-      <ft-icon-button
-        title="Change Video Formats"
-        theme="secondary"
-        icon="file-video"
-        :dropdown-names="formatTypeNames"
-        :dropdown-values="formatTypeValues"
-        @click="handleFormatChange"
-      />
-      <ft-icon-button
-        title="Share Video"
-        theme="secondary"
-        icon="share-alt"
-        :dropdown-names="shareNames"
-        :dropdown-values="shareValues"
-        @click="handleShare"
-      />
-    </ft-flex-box>
-    <p class="viewCount">
-      {{ parsedViewCount }}
-    </p>
-    <div class="likeBarContainer">
-      <div
-        class="likeBar"
-        :style="{ width: likePercentageRatio + '%' }"
-      />
-      <div class="dislikeBar" />
+    <div>
+      <p class="viewCount">
+        {{ parsedViewCount }}
+      </p>
+      <div class="likeBarContainer">
+        <div
+          class="likeSection"
+        >
+          <div
+            class="likeBar"
+            :style="{ background: `linear-gradient(to right, var(--accent-color) ${likePercentageRatio}%, #9E9E9E ${likePercentageRatio}%` }"
+          ></div>
+          <div>
+            <span class="likeCount">👍 {{ likeCount }}</span>
```

Comment:
I know I mentioned this in the chat room, but let's change these to fa-icons instead of using Emoji font.